plugins/connection/lxd.py

```diff
@@ -70,7 +82,13 @@ def exec_command(self, cmd, in_data=None, sudoable=True):
 
         self._display.vvv(u"EXEC {0}".format(cmd), host=self._host)
 
-        local_cmd = [self._lxc_cmd, "exec", self._host, "--", self._play_context.executable, "-c", cmd]
+        local_cmd = [
+            self._lxc_cmd,
+            "--project", self.get_option("project"),
+            "exec",
+            "%s:%s" % (self.get_option("remote"), self._host),
```

Comment:
Same for `remote`: does `local:<host>` behave the same as `<host>` for all versions of `lxc` supported by this plugin?

Reply:
Yeah, `local` has been a fixed remote in the LXD CLI ever since we've had the concept of remotes, so this shouldn't be a problem.

`--project` may be more problematic as LXD CLI prior to 3.6 will not know was a project is, similarly, a recent CLI interacting with an older LXD server will fail if that remote server is on a release prior to 3.6.

For this reason, it may be best to only pass `--project=` if the user specified a non-default project.